libs/api/src/lib/graphql/page/page.private-mutation.ts

```
@@ -465,76 +465,85 @@ export const updatePage = async (
-      }
-    },
-    include: fullPageInclude
-  })
+      },
+      include: fullPageInclude
+    })
+    return pageData
+  } catch (e) {
+    throw e
+    console.log(e.message)
+    console.log(e)
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)